mix.exs

```diff
@@ -2,26 +2,42 @@ defmodule Chess.Mixfile do
   use Mix.Project
 
   def project do
-    [app: :chess,
-     version: "0.0.1",
-     elixir: "~> 1.2",
-     elixirc_paths: elixirc_paths(Mix.env),
-     compilers: [:phoenix, :gettext] ++ Mix.compilers,
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     aliases: aliases(),
-     deps: deps()]
+    [
+      app: :chess,
+      version: "0.1.0",
+      elixir: "~> 1.5",
+      elixirc_paths: elixirc_paths(Mix.env),
+      compilers: [:phoenix, :gettext] ++ Mix.compilers,
+      build_embedded: Mix.env == :prod,
+      start_permanent: Mix.env == :prod,
+      aliases: aliases(),
+      deps: deps(),
+    ]
   end
 
   # Configuration for the OTP application.
   #
   # Type `mix help compile.app` for more information.
   def application do
-    [mod: {Chess, []},
-     applications: [:phoenix, :phoenix_pubsub, :phoenix_html, :cowboy, :logger,
-                    :gettext, :phoenix_ecto, :postgrex, :timex_ecto,
-                    :argon2_elixir, :comeonin, :formulator, :guardian,
-                    :elixir_make]]
+    [
+      mod: {Chess, []},
+      applications: [
+        :argon2_elixir,
+        :comeonin,
+        :cowboy,
+        :elixir_make,
+        :formulator,
+        :gettext,
+        :guardian,
+        :logger,
+        :phoenix,
+        :phoenix_ecto,
+        :phoenix_html,
+        :phoenix_pubsub,
+        :postgrex,
+        :timex_ecto,
+      ]
```

Comment:
I don't think you need the `applications` list here anymore with Elixir 1.5+. Instead, you can use `extra_applications: [:logger]` an the rest report themselves.